github_connector/demo/github_repository_branch.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
The file shold be suffixed with `_demo` at the end (`github_repository_branch_demo.xml`)